vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/IExpressionProcessor.java

```diff
@@ -20,5 +20,5 @@
 
 public interface IExpressionProcessor
 {
-    Optional<String> execute(String expression);
+    Optional<Object> execute(String expression);
 }
```

Comment:
```java
public interface IExpressionProcessor<T>
{
    Optional<T> execute(String expression);
}
```
?

Reply:
and in this case if you keep `UnaryExpressionProcessor` you won't need to change each and every expression processor